hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/replication/ECContainerHealthCheck.java

```diff
@@ -89,19 +96,33 @@ public ContainerHealthResult checkHealth(ContainerInfo container,
     return new ContainerHealthResult.HealthyResult(container);
   }
 
-  private ECContainerReplicaCount getReplicaCountWithPending(
+  private static ECContainerReplicaCount getReplicaCountWithPending(
```

Comment:
I guess whole method is not needed now, after the change in HDDS-6699 goes in - we are just passing the List<ContainerReplicaOp> through all the methods directly.

As an aside, we would not have needed to create a new instance of `ContainerReplicaPendingOps` here to form the list - we could have just created a list of ops directly. However I don't think we need any of this after we commit the above Jira, so its not important now.

Reply:
Yes, I agree, there is a overlap between, HDDS-6699. Let me wait on that.